src/Ryujinx/App.axaml.cs

```diff
@@ -111,5 +115,29 @@ private void ApplyConfiguredTheme()
                 ShowRestartDialog();
             }
         }
+
+        // Convert PlatformThemeVariant to the expected ThemeVariant type
+        private ThemeVariant ConvertThemeVariant(PlatformThemeVariant platformThemeVariant)
+        {
+            switch (platformThemeVariant)
+            {
+                case PlatformThemeVariant.Dark:
+                    return ThemeVariant.Dark;
+
+                case PlatformThemeVariant.Light:
+                    return ThemeVariant.Light;
+
+                default:
+                    return ThemeVariant.Default;
+            }
+        }
+
+        private ThemeVariant DetectSystemTheme()
+        {
+            var platformSettings = this.PlatformSettings;
+            var colorValues = platformSettings.GetColorValues();
```

Comment:
```suggestion
            var colorValues = PlatformSettings.GetColorValues();
```
And the `platformSettings` variable can be removed then. Or is there a special reason for having this variable?

Reply:
Yeah, my bad. Just `PlatformSettings` is enough.